src/main/java/io/vertx/cassandra/impl/CassandraClientImpl.java

```diff
@@ -70,7 +74,35 @@ public CassandraClient connect(String keyspace, Handler<AsyncResult<Void>> conne
       }
     }
 
-    Cluster build = builder.withPort(options.port()).build();
+    Cluster build = builder.withNettyOptions(
+      new NettyOptions() {
```

Comment:
Can you move this to an inner class which just takes the `vertx` instance as constructor argument?

Reply:
Yes, just done